src/shared/components/Tooltip/Tooltip.style.ts

```diff
@@ -0,0 +1,49 @@
+import styled from '@emotion/styled'
+import { colors, sizes, typography, zIndex, transitions } from '@/shared/theme'
+
+type TooltipProps = {
+  isActive?: boolean
+  arrowDisabled?: boolean
+}
+
+export const StyledTooltip = styled.div<TooltipProps>`
+  position: relative;
+  display: inline-block;
+  &::before {
+    position: absolute;
+    display: inline-block;
+    content: attr(data-text);
+    bottom: 0;
+    transform: translateY(calc(100% + 10px)) scale(${({ isActive }) => (isActive ? 1 : 1.1)});
+    min-width: 100px;
+    max-width: 200px;
+    opacity: ${({ isActive }) => (isActive ? 1 : 0)};
+    font-size: ${typography.sizes.caption};
+    font-weight: ${typography.weights.thin};
+    padding: ${sizes(2.5)} ${sizes(2)};
+    background: ${colors.gray[400]};
+    color: ${colors.white};
+    z-index: ${zIndex.nearOverlay};
+    transition: transform 100ms ${transitions.easing}, opacity 100ms ${transitions.easing};
```

Comment:
```suggestion
    transition: transform 100ms ${transitions.easing}, opacity 100ms ${transitions.easing};
    line-height: 16px; 
```
I know that in the design there is no multiline tooltip, but it just feels better with slightly increased line-height,